projects/adxrs290-pmdz/src/platform/stm32/parameters.c

```diff
@@ -51,6 +51,5 @@ struct stm32_spi_init_param adxrs290_spi_extra_ip  = {
 
 struct stm32_gpio_init_param adxrs290_gpio_extra_ip = {
 	.mode = GPIO_MODE_INPUT,
-	.port = GPIOA,
```

Comment:
In adxrs290 project, please add .port = 0 explicitly.
It's very weird to select GPIOA implicitly, without initializing .port.

Reply:
GPIOA is selected in parameters.h (https://github.com/analogdevicesinc/no-OS/blob/18d5dc15506bd3c41eec5f030a22039bdac6a264/projects/adxrs290-pmdz/src/platform/stm32/parameters.h#L73) as a member of adxrs290_gpio_sync_ip (https://github.com/analogdevicesinc/no-OS/blob/96f4a44948128aecf687343973d0e53470da0c6d/projects/adxrs290-pmdz/src/common/common_data.c#L59).

Reply:
ok, i missed it

Reply:
Will also set SPI_CS_PORT pins correctly